source/PreferencesPanel.cpp

```diff
@@ -781,6 +781,9 @@ void PreferencesPanel::DrawSettings()
 		"Always underline shortcuts",
 		REACTIVATE_HELP,
 		"Interrupt fast-forward",
+		"'Sell Outfits' without outfitter",
+		"Confirm 'Sell Outfits' button",
+		"Confirm 'Sell Flotsam' button",
```

Comment:
These can probably have their own category. "Trading" or "Trading panel" or something like that maybe?